routes/web.php

```diff
@@ -501,7 +505,7 @@ function()
     Route::get('authority/search', 'AuthorityController@search');
     Route::resource('authority', 'AuthorityController');
     Route::resource('sketchbook', 'SketchbookController');
-    Route::resource('slide', 'SlideController');
+    Route::resource('notices', 'NoticeController');
```

Comment:
```suggestion
    Route::resource('slide', 'SlideController');
    Route::resource('notices', 'NoticeController');
```

(I guess this was done unintentionally)

Reply:
Great catch, thanks!